service/controller/v24/resource/ipam/create.go

```diff
@@ -212,6 +219,63 @@ func (r *Resource) selectRandomAZs(n int) ([]string, error) {
 	return shuffledAZs, nil
 }
 
+// getAvailabilityZonesForInstanceTypes gets availability zones for current
+// region and removes ones that don't have required instance types present.
+func getAvailabilityZonesForInstanceTypes(ctx context.Context, instanceTypes ...string) ([]string, error) {
+	ctlCtx, err := controllercontext.FromContext(ctx)
```

Comment:
Should be `cc`:https://github.com/giantswarm/fmt/blob/master/go/variable_names.md

Reply:
Ah, yeah. I was wondering which way did we decide. I already tripped once on confusing `cc` & `cr` in same function. Oh well, I'll switch :)